OrbitGl/FrameTrack.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_DELTA_TRACK_H_
+#define ORBIT_GL_DELTA_TRACK_H_
+
+#include "TimerTrack.h"
+
+class FrameTrack : public TimerTrack {
+ public:
+  explicit FrameTrack(TimeGraph* time_graph, const orbit_client_protos::FunctionInfo& function);
+  [[nodiscard]] Type GetType() const override { return kDeltaTrack; }
```

Comment:
Should kDeltaTrack be named kFrameTrack instead?

Reply:
Yes, forgot the rename this.